File: 加密算法/加密算法/ViewController.m
//
//  ViewController.m
//  加密算法
//
//  Created by 侨品汇 on 2017/11/14.
//  Copyright © 2017年 陈博. All rights reserved.
//

#import "ViewController.h"
#import "DesEncryptionManager.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    NSLog(@"加密后的结果:%@",[DesEncryptionManager encryptUseDes:@"123" key:@"hew"]);
}



@end

File: Swift-IM/Client/Client/ViewController.swift
//
//  ViewController.swift
//  Client
//
//  Created by 陈博 on 2018/4/7.
//  Copyright © 2018年 陈博. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    fileprivate lazy var client : ClientSocket = ClientSocket(addr: "192.168.10.101", port: 7878)
    

    override func viewDidLoad() {
        super.viewDidLoad()
        
    }
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        if client.connectServer(){
            print("!!!!!")
        }
    }
}

